### PR TITLE
Updates to reflect D3D12TranslationLayer refactor

### PR DESCRIPTION
**Pull request by vdwtanner**

### CODE REVIEW

**Review by jenatali — APPROVED** [review]

**Comment by jenatali** [comment]
We should update the SHA once the `D3D12TranslationLayer` changes land.

**Comment by vdwtanner** [comment]
We currently have the cmake point at the tip of master, so as long as we hold off on merging this till d3d12transaltionlayer is updated it should be fine

**Pull request merged**
